www/js/homepage-anon.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Created by skuroda on 11/6/15.
+ */
+var db;
+
+$(document).ready(function(){
+    // Consider change to the devel site url.
+    var DOMAIN = "http://master-caterpillars.vipapps.unc.edu";
+    var $list_length = $(".survey_item").length; //computes number of items in the survey list
+
+    document.addEventListener("deviceready", onDeviceReady, false);
+    //Return to start screen if android back button is pressed
+    function onDeviceReady(){
+        document.addEventListener("backbutton", function(e){
+            e.preventDefault();
+            window.location.assign("StartScreen.html");
+        }, false);
+        
+        db=window.sqlitePlugin.openDatabase(
+        {name: 'app.db', location: 'default'}, 
```

Comment:
Is there a reason to open db at homepage? I think you only need to open it for survey page.